internal/cloudapi/v2/handler.go

```diff
@@ -1125,11 +1129,25 @@ func (h *apiHandlers) getComposeMetadataImpl(ctx echo.Context, id string) error
 		})
 	}
 
-	if result.OSBuildOutput == nil || len(result.OSBuildOutput.Log) == 0 {
+	if err != nil {
```

Comment:
I think this condition will be always `false`, `err` is already checked on line 1106.